src/2_RTB/NormeDiProgetto/content.typ

```diff
@@ -685,25 +902,62 @@ Per identificare  le metriche si usa la seguente formattazione:
+
+Eventuali variazioni nella distribuzione dei ruoli, rispetto alla pianificazione iniziale, vengono documentate e giustificate nel *Consuntivo* del _Piano di Progetto v1.0_. Questo approccio consente al team di adattarsi in modo flessibile alle esigenze emergenti, mantenendo costantemente un quadro chiaro delle variazioni e delle ragioni che sottendono a tali modifiche.
+
+==== Documentazione Fine Sprint
+
+Il Responsabile è incaricato di sviluppare un preventivo dettagliato per le successive due settimane subito dopo la conclusione dello sprint attuale, fornendo una guida chiara al team su aspettative e obiettivi.
+
+Nel contesto della gestione dello sprint, il Responsabile si impegna anche a compilare un consuntivo dettagliato dello sprint appena concluso. Si analizzano le attività svolte, i tempi impiegati rispetto alle stime previste e qualsiasi deviazione dai piani originali. Tale consuntivo rappresenta una preziosa fonte di apprendimento per il team, consentendo di identificare aree di miglioramento e di ottimizzare la pianificazione futura.
+
+Per garantire un monitoraggio efficace delle prestazioni, il Responsabile utilizza e calcola le metriche, presenti nel documento _Piano di Qualifica v1.0_, rispetto allo sprint appena concluso, valutando l'efficacia del processo e apportando eventuali correzioni o miglioramenti per il successo continuo del progetto.
+
+==== Consuntivo   
+
+È stato sviluppato uno script in Python, integrato con Typst, per semplificare la generazione automatica di tabelle e grafici relativi al consuntivo di ogni singolo sprint del progetto. L'automazione coinvolge l'estrazione dei dati dal foglio di calcolo condiviso, dove vengono registrate le ore produttive del team. I dati vengono successivamente organizzati per sprint e archiviati in file CSV.
+
+L'elaborazione comprende la creazione di un DataFrame consolidato che rappresenta le ore lavorate per ogni ruolo e membro del team durante uno sprint, includendo anche i costi associati. I risultati di questa analisi vengono salvati in un file CSV.
+
+Per visualizzare i dati aggiornati relativi a ciascuno sprint, è possibile utilizzare le seguenti funzioni all'interno del file:
+
+\#rendicontazioneOreAPosteriori(sprintNumber: "number")
+\#rendicontazioneCostiAPosteriori(sprintNumber: "number")
+
+Dove "number" rappresenta il numero specifico dello sprint di interesse. Queste funzioni consentono di ottenere report aggiornati sulla distribuzione delle ore per ruolo e persona, nonché sui costi associati, semplificando la gestione.
+
```

Comment:
rivedi in generale la struttura di sta parte, forse va sotto la parte dei ruoli del responsabile

Reply:
per quanto riguarda il cosuntivo invece va bene va bene